cg/meta/workflow/fluffy.py

```diff
@@ -255,7 +256,7 @@ def get_sample_sheet_housekeeper_path(self, flowcell_name: str) -> Path:
         Returns the path to original samplesheet file that is added to Housekeeper
         """
         sample_sheet_query: list = self.housekeeper_api.files(
-            bundle=flowcell_name, tags=["samplesheet"]
+            bundle=flowcell_name, tags=[SequencingFileTag.SAMPLE_SHEET]
```

Comment:
Same concern as above with the tags. My hunch is that it is out of scope for this PR even if we should consider archived sample sheets, though.

Reply:
fixed!